assessment_software/src/app/help-component/help-component.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component, Input, HostListener } from '@angular/core';
+
+@Component({
+  selector: 'app-help-component',
+  templateUrl: './help-component.component.html',
+  styleUrls: ['./help-component.component.css'],
+})
+export class HelpComponentComponent {
+  hide = true;
```

Comment:
on hover, it should show an explanation dialog. It does not. what happens instead is that the dialogue only appears on mouse leave

Reply:
resolved.